admin_guide/ipsec.adoc

```diff
@@ -37,21 +37,21 @@ with a host that does.
 == Encrypting Hosts
 
 [[admin-guide-ipsec-prerequisites]]
-=== Step 1: Prerequisites
+=== Prerequisites
 
-At this time, *libreswan* version 3.15 is the latest version supported on Red
-Hat Enterprise Linux 7. Ensure that *libreswan* 3.15 or later is installed on
+* Ensure that *libreswan* 3.15 or later is installed on
 cluster hosts. If
 xref:../admin_guide/ipsec.adoc#admin-guide-ipsec-opportunistic-group-configuration[opportunistic
 group functionality] is required, then *libreswan* version 3.19 or later is
 required.
-
-xref:../install_config/configuring_sdn.adoc#install-config-configuring-sdn[Configure the SDN]
-MTU to allow space for the IPSec header. In the configuration described here
-IPSec requires 62 bytes. If the cluster is operating on an ethernet network with
-an MTU of 1500 then the SDN MTU should be 1388, to allow for the overhead of
-IPSec and the SDN encapsulation.
-
+* See the
+ xref:../install_config/configuring_sdn.adoc#configuring-the-pod-network-on-nodes[Configure
+ the pod network on nodes] section for information on how to configure the MTU
+ to allow space for the IPSec header. This topic describes an IPSec
+ configuration that requires 62 bytes. If the cluster is operating on an
+ ethernet network with an MTU of 1500 then the SDN MTU should be 1388, to allow
+ for the overhead of IPSec and the SDN encapsulation.
++
```

Comment:
I think ethernet should be initial-capped: Ethernet. Style guide says to cap it; but our docs are inconsistent. 7 instances capped; 8 lower-case.
https://mojo.redhat.com/docs/DOC-28115#e

Reply:
@mburke5678 Checks out:

https://www.reddit.com/r/grammar/comments/34eldl/capitalize_ethernet/

To stick with IBM I'll capatalize. Good eye!